samples/rag-cosmosdb/SemanticCosmosDBSearchEmbeddings/FilePrompt.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
please add README in rag-cosmosdb folder on how to configure cosmos db, update settings and use it with Semantic Search binding, you can refer AI Search binding README